### PR TITLE
[9.0][MIG] stock_warehouse_orderpoint_stock_info

### PR DESCRIPTION
**Pull request by JordiBForgeFlow**
Migration to version 9.0

### CODE REVIEW

**Comment by JordiBForgeFlow** [comment]
@mreficent @gmeficent can you review?

**Comment by JordiBForgeFlow** [comment]
@oihane can you review?

File: stock_warehouse_orderpoint_stock_info/views/stock_warehouse_orderpoint_view.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <openerp>
```
Comment (andhit-r) [inline stock_warehouse_orderpoint_stock_info/views/stock_warehouse_orderpoint_view.xml]: Use <odoo> tag

**Review by andhit-r — CHANGES_REQUESTED** [review]
I haven't test functionality on runbot since it's still red. I will review it again when runbot is green. 

PS:
Why there are changes on PO files on your last commit?

**Comment by oihane** [comment]
There is a problema with views, that makes it fail, @jbeficent can you check it

File: stock_warehouse_orderpoint_stock_info/views/stock_warehouse_orderpoint_view.xml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+        <record model="ir.ui.view" id="stock_warehouse_orderpoint_search_view">
+            <field name="name">stock.warehouser.orderpoint.search</field>
+            <field name="model">stock.warehouse.orderpoint</field>
+            <field name="inherit_id" ref="stock.warehouse_orderpoint_search" />
+            <field name="arch" type="xml">
+                <field name="product_id" position="after">
+                    <field name="product_category" filter_domain="[('product_category','child_of',self)]"/>
+                </field>
+                <xpath expr="//group[0]" position="inside">
```
Comment (andhit-r) [inline stock_warehouse_orderpoint_stock_info/views/stock_warehouse_orderpoint_view.xml]: group[1]. XML index start from 1

**Comment by MiquelRForgeFlow** [comment]
Thanks @andhit-r. It's all green now.

PS: And without the PO changes.

**Review by andhit-r — APPROVED** [review]
:+1:

**Review by oihane — APPROVED** [review]
LGTM :+1:

File: stock_warehouse_orderpoint_stock_info/models/stock_warehouse_orderpoint.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 OdooMRP Team
+# Copyright 2016 AvanzOSC
+# Copyright 2016 Pedro M. Baeza <pedro.baeza@tecnativa.com>
+# Copyright 2016 Serpent Consulting Services Pvt. Ltd.
+# Copyright 2016 Eficent Business and IT Consulting Services, S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import models, fields, api
```
Comment (rousseldenis) [inline stock_warehouse_orderpoint_stock_info/models/stock_warehouse_orderpoint.py:9]: api, fields, models

**Comment by JordiBForgeFlow** [comment]
Thanks to all for your review!

File: stock_warehouse_orderpoint_stock_info/models/stock_warehouse_orderpoint.py [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+        compute='_compute_product_available_qty')
+    outgoing_location_qty = fields.Float(
+        string='Outgoing On Location',
+        compute='_compute_product_available_qty')
+    virtual_location_qty = fields.Float(
+        string='Forecast On Location',
+        compute='_compute_product_available_qty')
+    product_category = fields.Many2one(string='Product Category',
+                                       related='product_id.categ_id',
+                                       store=True)
```
Comment (rousseldenis) [inline stock_warehouse_orderpoint_stock_info/models/stock_warehouse_orderpoint.py:40]: Shouldn't it be readonly (In this case, a write is done on product.category)

**Comment by rousseldenis** [comment]
Oups, too late !

**Pull request merged**